### ISSUE TITLE
3.0: Instance isn't starting with Bind mounts

### ISSUE THREAD

**Issue by jmstover** [apptainer/singularity#1989]
## Version of Singularity:

3.0.0-alpha.1-347-g3b2f9ac

### Expected behavior
```
$ singularity instance start -B web/htdocs:/usr/local/apache2/htdocs:ro -B web/logs:/usr/local/apache2/logs httpd.sif httpd
$ ps auxwwf | grep 'instance' | grep -v 'grep'
jason    27053  0.0  0.0 328628 10452 ?        Ssl  14:24   0:00 Singularity instance: jason [httpd]
```

### Actual behavior
```
$ singularity instance start -B web/htdocs:/usr/local/apache2/htdocs:ro -B web/logs:/usr/local/apache2/logs httpd.sif httpd
$ ps auxwwf | grep 'instance' | grep -v 'grep'
$

# No debug output
$ singularity -d instance start -B web/htdocs:/usr/local/apache2/htdocs:ro -B web/logs:/usr/local/apache2/logs httpd.sif httpd
$
```

### Steps to reproduce behavior

Start an instance *with* a Bind mount. If a user bind mount is _not_ specified, the instance is started. But with a bind mount (`-B` option) it is not started.

**Comment by jmstover** [comment]
Dug into it more, and the instance is _not starting_ when a Bind mount is specified, not just not writing out the state information. So, redone.

**Comment by cclerget** [comment]
Fixed by #1996
